VMEncryption/main/EncryptionSettingsUtil.py

```diff
@@ -31,7 +33,8 @@ class EncryptionSettingsUtil(object):
 
     def __init__(self, logger):
         self.logger = logger
-        self._CURRENT_DISK_ENCRYPTION_DATA_VERSION = "3.0"
+        self._CURRENT_DISK_ENCRYPTION_DATA_VERSION = "4.0"
+        self._PREVIOUS_DISK_ENCRYPTION_DATA_VERSION = "3.0"
```

Comment:
how about explicitly naming the variable to correspond to the version "_DISK_ENCRYPTION_DATA_VERSION_V3" and "_DISK_ENCRYPTION_DATA_VERSION_V4" ?

Reply:
Good idea- will do